LICENSE

```diff
@@ -178,15 +178,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
```

Comment:
Apache changed the text, and you took a fresh copy, I assume?

Reply:
The copy with `[]` is GitHub's version. The copy with `{}` is our version which we have in the operators repository.